FEAT : detail화면 추가

leehandsub opened the pull request.

leehandsub committed:
refact : 코드 정리

- 함수가 한개의역할만 하게
- 코틀린 함수 사용 isblank
- retrofit를 굳이 새로 저장할일 없기 때문에 val로 변경

leehandsub committed:
chore : parcelable 사용을 위한 플러그인 추가

leehandsub committed:
feat : detail화면 구성

leehandsub committed:
chore : 화면 이동을 위한 navigation 라이브러리 추가

leehandsub committed:
feat : 화면 전환

- 화면 전환
- 전환하면 photo 데이터 넘기기

rlarlvy153 commented on app/src/main/java/com/example/simplegallery/ApplicationClass.kt:
```diff
@@ -9,25 +9,23 @@ import java.util.concurrent.TimeUnit
-            .connectTimeout(5000, TimeUnit.MILLISECONDS)
-            .addInterceptor(loggingInterceptor)
-            .build()
+        val retrofit: Retrofit by lazy {
```
얘가 왜 앱에 속함?

rlarlvy153 commented on app/src/main/java/com/example/simplegallery/MainActivity.kt:
```diff
@@ -26,10 +33,29 @@ class MainActivity : ComponentActivity() {
+fun Navigation(photoListViewModel: PhotoListViewModel) {
+    val navController = rememberNavController()
+
+    NavHost(navController = navController, startDestination = "screen1") {
```
네이밍같은거 숫자붙이는형태로 하지말고 구체적으로 하기

leehandsub committed:
refact : 네이밍 이상한거 수정

leehandsub committed:
refact : retrofit에 대한 위치 고민

- 어차피 전역으로 쓸건데 applicationclass나 전역이나 똑같다.
- 혹시 앱최초 실행이 느리다면 다르게 retrofit객체를 좀 더 늦게 생성하게 할 것이다.
